tailwind.config.js

```diff
@@ -3,7 +3,23 @@ export default {
   content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
   darkMode: "selector",
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        headerBrand: "#003049",
+        headerBackground: "#fdf0d5",
+      },
+      borderRadius: {
+        "custom-xs": "4px",
+        "custom-sm": "8px",
+        "custom-md": "16px",
+        "custom-lg": "24px",
+        "custom-xl": "32px",
+        "custom-2xl": "40px",
+        "custom-3xl": "48px",
+        "custom-full": "9999px", // For fully rounded/circular corners
+        "custom-circle": "50%", // For a perfect circle
```

Comment:
Tbh, I don't like these customisations. Can we please stick to what tailwind offers by default 😅

Reply:
i can remove these, these are just extended styles of tailwind most of them are same with official tailwind, should i remove?

Reply:
Yes please!